mls-rs-crypto-webcrypto/src/ec/ecdh.rs

```diff
@@ -165,6 +165,10 @@ impl DhType for Ecdh {
     fn public_key_validate(&self, _key: &HpkePublicKey) -> Result<(), Self::Error> {
         Ok(())
     }
+
+    fn public_key_size(&self) -> usize {
+        self.curve.secret_key_size()
```

Comment:
Is that right or should it be as below?
```suggestion
        self.curve.public_key_size()
```